Sources/SwiftFormat/API/Configuration.swift

```diff
@@ -415,7 +415,7 @@ public struct Configuration: Codable, Equatable {
       if FileManager.default.isReadableFile(atPath: candidateFile.path) {
         return candidateFile
       }
-    } while candidateDirectory.path != "/"
+    } while candidateDirectory.pathComponents.count > 1
```

Comment:
The concern that I have with this on Windows is that this would return `true` for `C:` and `\`, both of which are relative paths.